src/annotator/test/guest-test.js

```diff
@@ -220,6 +232,29 @@ describe('Guest', () => {
       });
     });
 
+    describe('on "scrollToClosestOffScreenAnchor" event', () => {
+      it('calls "Guest#._scrollToClosestOffScreenAnchor"', () => {
```

Comment:
The test description references an internal method, although the test itself only checks interaction with external classes, which is the right thing to do. I would suggest to reword this description as "scrolls to the nearest anchor in the given set" or similar.

Reply:
👍🏼